### DISCUSSION TITLE
Wrong redirect_uri after domain change

### DISCUSSION THREAD

**Discussion by msebi** [vercel/vercel#8831] [Help] [answered]
In a deployment the domain name is of the following form:

```
https://<proj-name>-<hash>-<git-user>.vercel.app/
```

in the web UI, there are two more domains assigned to the same deployment:

```
https://<meaningful-name>.vercel.app/
```

and another similar to the first but with a different hash value. 

all domains show the correct deployment (nextjs app). However, the app uses auth0; 

when going to the login page, the following is shown:

```
Callback URL mismatch.
The provided redirect_uri is not in the list of allowed callback URLs.
Please go to the [Application Settings page] and make sure you are sending a valid callback url from your application.
```

but the redirect_uri points to a deployment that has already been deleted. This looks like a bug in vercel. 

Is there a workaround? 

**Edit:**

The url in an env var had to be changed ...

**Comment by amyegan** [comment, marked as answer]
Hi @msebi! Are you still facing this problem? Or did updating the environment variable fix it for you?

**Reply by msebi** [reply]
Hi Amy!

Yep, the update fixed it :)
